templates/up.sh.j2

```diff
@@ -1,13 +1,15 @@
 #!/bin/sh
 # {{ ansible_managed }}
 set +e
-brctl addbr {{ openvpn_bridge_dev }}
+brctl addbr {{ openvpn_bridge_name }}; true
 if [ $? != 0 ]; then
 	sleep 2
 	exec systemctl restart openvpn.service
 fi
-brctl addif {{ openvpn_bridge_dev }} {{ openvpn_dev }}; true
-brctl addif {{ openvpn_bridge_dev }} {{ openvpn_interface }}; true
-ip link set {{ openvpn_bridge_dev }} up;
-ip link set {{ openvpn_dev }} up;
-ip address replace {{ openvpn_bridge.address }} dev {{ openvpn_bridge_dev }}; true
+brctl addif {{ openvpn_bridge_name }} tap0; true
```

Comment:
Please use the openvpn environment or argument variables

Reply:
Done.